charts/opentelemetry-demo/templates/_objects.tpl

```diff
@@ -83,6 +97,21 @@ spec:
     {{- include "otel-demo.selectorLabels" . | nindent 4 }}
 {{- end}}
 {{- end}}
+{{- define "otel.demo.configmap" }}
```

Comment:
use otel-demo instead of otel.demo for the name here to stay consistent with other names.

```suggestion
{{- define "otel-demo.configmap" }}
```